_posts/devices/2019-01-07-xdk-driver.md

```diff
@@ -0,0 +1,97 @@
+- three-axis gyroscope (or four-axis rotecion-quaternion)
+- light
+- noise
+- pressure
+- ambient temperature
+- humidity
+- sd-card detect status
+- push buttons
+- three-axis magnetic field
+- magnetometer resistence
+- led status
+- rms voltage of LEM sensor
+
+| Resource | Unit | Description |
+|----------|------|-------------|
+| **ACCELERATION_X**, **ACCELERATION_Y**, **ACCELERATION_Z** | G | The proper acceleration for each axis* |
+| **GYROSCOPE_X**, **GYROSCOPE_Y**, **GYROSCOPE_Z** | rad/S | The angular acceleration for each axis* |
+| **LIGHT** | lux | The light value |
+| **NOISE** | DpSpl | The acustic pressure value |
+| **PRESSURE** | Pa | The pressure value |
+| **TEMPERATURE** | °C | The temperature value |
+| **HUMIDITY** | %rH | The relative humidity |
+| **SD_CARD_DETECTION_STATUS** | boolean | SD-Card detect status |
+| **PUSH_BUTTONS** | bit | Button status, encoded as bit field: Bit 0, Button 1; Bit 1, Button 2 |
+| **MAGNETOMETER_X**, **MAGNETOMETER_Y**, **MAGNETOMETERE_Z** | uT | The magnetometer value for each axis |
+| **MAGNETOMETER_RESISTENCE** | ohm | The magnetometer resistence value |
+| **LED_STATUS** | bit | Led status, encoded as a bit field: Bit 0: yellow Led; Bit 1: orange Led; Bit 2: red Led|
+| **VOLTAGE_LEM** | mV | RMS voltage of LEM sensor |
+
+*If enable Quaternion then:
+
+| Resource | Unit | Description |
+|----------|------|-------------|
+|**QUATERNION_M**, **QUATERNION_X**, **QUATERNION_Y**, **QUATERNION_Z** | number | The rotation-quaternion for each axis |
+
+
+When a notification is enabled for a specific channel (sensor), the notification period can't be set. Use the Timer in Wire Graph to set notification period.
+
+## Documentation
+
+All information on the Xdk110 is in Xdk Bosch Connectivity [here](https://xdk.bosch-connectivity.com/home). XDK-Workbench is tool to develop software for Xdk based on Eclipse platform, for XDK-Workbench download [here](https://xdk.bosch-connectivity.com/it/software-downloads). XDK-Workbench is required to install VirtualXdkDemo.
+
+{% include alerts.html message='XDK-Workbench version 3.6.0 have a problem, i raccomend installing the version 3.5.0' %}
+
+All information on the VirtualXdkDemo is in Virtual XDK application user guide [here](http://xdk.bosch-connectivity.com/xdk_docs/html/_x_d_k__v_i_r_t_u_a_l__x_d_k__a_p_p__u_s_e_r__g_u_i_d_e.html), in  Virtual XDK application user guide you will find the references to the UUID and data format. 
+
+{% include alerts.html message='In Virtual XDK application user guide for Xdk API version 3.4.0 a Characteristic of a Service is missing. For Control XDK Service the Characteristic: 55b741d5-7ada-11e4-82f8-0800200c9a66 is missing. This Characteristic is used to enable or disable the Rotations Quaternions. If you write 0x00 then enable Accelemoter and Gyroscope, else you write 0x01 then enable Rotaton Quaternions.' %}
+
+if enable Rotation Quaternions  then the interpretation of the data is:
```

Comment:
Is this piece of information useful for the user?

If so, please review "if enable Rotation Quaternions  then the interpretation of the data is:" and change it to something like: "If quaternion representation is enabled, the data format is as follows:"